app/uk/gov/hmrc/preferenceschangednotifier/controllers/model/RestInstantFormat.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.preferenceschangednotifier.controllers.model
+
+import play.api.libs.json.{
+  Format,
+  JsSuccess,
+  JsError,
+  JsResult,
+  JsValue,
+  JsString,
+  Reads,
+  Writes
+}
+
+import java.time.Instant
+import scala.util.Try
+
+object RestInstantFormat {
+  implicit val format = Format(restInstantReads, restInstantWrites)
+
+  private lazy val restInstantReads: Reads[Instant] =
```

Comment:
Do we have to write this Reads?

Reply:
Hi @muralishub , thanks for catching, have made this a read only along with the type that uses it.